tap_mambu/sync.py

```diff
@@ -105,7 +105,7 @@ def process_records(catalog, #pylint: disable=too-many-branches
                         last_dttm = transform_datetime(last_datetime)
                         bookmark_dttm = transform_datetime(transformed_record[bookmark_field])
                         # Keep only records whose bookmark is after the last_datetime
-                        if bookmark_dttm >= last_dttm:
+                        if bookmark_dttm > last_dttm:
```

Comment:
I think we'll want to keep this as `>=` to account for multiple records being synced at the same timestamp.

e.g. if there are 10 records `r` at timestamp `t=2` and on the first sync, we sync through only the first 4 of them, we want to ensure that those remaining 6 records aren't skipped over at the cost of us having to redundantly sync the first 4.

Reply:
Good catch! My intent was to avoid re-syncing previously synced records on subsequent runs of the tap. But I missed that the `write_bookmark` function was updating the bookmark within the while loop that is batch processing records. What do you think about moving the `write_bookmark` function to the end of `sync_endpoint` after the batch processing has finished? I just moved it and tested it out it and it looks to work just fine that way as well. I updated the branch with the change. Let me know if that is acceptable.

Reply:
Yep, I like that.

👍  sounds good to me.